### PR TITLE
feat(events): Add update_connection

### PR DESCRIPTION
**Pull request by GonzaloSaad**
Work Done
---
Add `update_connection` to Response & Backend in Events

Tests
---
- [x] `TestAccAWSCloudWatchEventConnection_invocationHttpParameters`

image

Test suite completed :partying_face:

### CODE REVIEW

**Review by bblommers — CHANGES_REQUESTED** [review]
Hi @GonzaloSaad, thanks for the PR! Would you mind adding this suite to our whitelist of Terraform tests? (tests/terraform-tests.success.txt)
That way this specific test will run with every build, and we can be sure this keeps working/doesn't break further down the line

File: tests/terraform-tests.success.txt [hunk 1]
```diff
@@ -51,4 +52,4 @@ TestAccAWSSageMakerPrebuiltECRImage
 TestAccAWSSsmParameterDataSource
 TestAccAWSUserGroupMembership
 TestAccAWSUserPolicyAttachment
-TestAccAWSUserSSHKey
\ No newline at end of file
+TestAccAWSUserSSHKey
```
Comment (GonzaloSaad) [inline tests/terraform-tests.success.txt:55]: Newline maybe?
image

**Comment by GonzaloSaad** [comment]
> Hi @GonzaloSaad, thanks for the PR! Would you mind adding this suite to our whitelist of Terraform tests? (tests/terraform-tests.success.txt)
> That way this specific test will run with every build, and we can be sure this keeps working/doesn't break further down the line

Done! Will do it for the next suites! Thank you very much for the feedback, and nice to start working with you!

**Review by bblommers — APPROVED** [review]
LGTM! Thanks for adding it to the list @GonzaloSaad 👍

**Pull request merged**